app/views/listings/_details.html.erb

```diff
@@ -0,0 +1,173 @@
+<div role="tabpanel" class="tab-pane fade show active" id="tab-1">
+    <div class="row">
+        <div class="col">
+            <div class="row">
+                <div class="col">
+                    <h3>Description</h3>
+                    <hr />
+                    <p class="MontserratFont">
+                    <em> <strong> House Type: </strong> </em> &nbsp <%=  @listing.home_type %>  <br />
+                    <em> <strong> Room Type: </strong> </em> &nbsp <%=  @listing.room_type %> 
+                    </p>
+                    <p class="MontserratFont"> <%= @listing.listing_description%> </p>                    
+                </div>
+            </div>
+            <div class="row" style="margin-top: 20px;">
+                <div class="col">
+                    <h3>Details</h3>
+                    <hr />
+                    <div class="row MontserratFont">
+                        <div class="border rounded border-dark amenitiesIcons">
+                            <i class="fa fa-home d-xl-flex justify-content-xl-center fa-3x" style="color: rgb(0,0,0);margin-top: 10px;"></i>
+                            <p class="text-center">
+                            <%= @listing.home_type %></p>
+                        </div>
+                        <div class="border rounded border-dark amenitiesIcons">
+                            <i class="fa fa-user d-xl-flex justify-content-xl-center fa-3x" style="color: rgb(0,0,0);margin-top: 10px;"></i>
+                            <p class="text-center">
+                            <%= case @listing.accomodate
+                                when 1
+                                    "1 Guest"
+                                when 2
+                                    "2 Guests"
+                                when 3
+                                    "3 Guests"
+                                when 4
+                                    "4+ Guests"
+                                end
+                                %></p>
```

Comment:
Simplify this by using the following:
`<%= pluralize(@room.accomodate, "Guest") %>`

This will display the number of guests and display "Guest" as plural or singular depending on the `@room.accomodate`. 

Please do the same with bedroom count and bathroom count.